components/post_view/post_time/post_time.jsx

```diff
@@ -45,17 +52,35 @@ export default class PostTime extends React.PureComponent {
         };
     }
 
+    getCurrentUserTimezone = () => {
+        const userId = UserStore.getCurrentId();
+        const timezone = UserStore.getTimezone(userId);
+        return getCurrentTimezone(timezone);
+    };
+
     renderTimeTag() {
+        const userTimezone = this.getCurrentUserTimezone();
         const date = new Date(this.props.eventTime);
-        const militaryTime = this.props.useMilitaryTime;
 
-        const hour = militaryTime ? date.getHours() : (date.getHours() % 12 || 12);
-        let minute = date.getMinutes();
-        minute = minute >= 10 ? minute : ('0' + minute);
-        let time = '';
+        // const militaryTime = this.props.useMilitaryTime;
```

Comment:
This commented lines can be removed because the military time is already solved by the toLocaleString.

Reply:
👍